src/main/scala/beam/agentsim/infrastructure/parking/ParkingZoneSearch.scala

```diff
@@ -105,6 +105,25 @@ object ParkingZoneSearch {
     }
   }
 
+  def canThisCarParkHere(
```

Comment:
This method does not appear to be used. Is this PR a WIP maybe?

Reply:
Used at line 89 so that the search only returns stalls that are both unoccupied and have the correct charget status. Going to still go back to marking this a draft though because merging these changes into production was a bit tricky and if the changes from there are going into Develop it might make sense to re-do these changes in a branch based off of that.

Reply:
D'oh - I swear I did a find and it didn't pop up before...nvm...sorry about that. That said, not going to touch until it's ready :)